v_4_5_0/files/conf/k8s-addons

```diff
@@ -14,11 +14,16 @@ NAMESPACES="default kube-system"
 for namespace in ${NAMESPACES}
 do
     while
```

Comment:
I think this should become `if`.

Reply:
I think we should stop using bash magic where possible. 🙈

Reply:
It is while for the network glitches case, when `get namespaces` fails just because of api being unresponsive

Reply:
Does it matter? `$?` will be non-zero anyway.